src/ucp/core/ucp_worker.c

```diff
@@ -642,7 +642,9 @@ ucs_status_t ucp_worker_arm(ucp_worker_h worker)
         wakeup = worker->wakeup.iface_wakeups[tl_id];
         if (wakeup != NULL) {
             status = uct_wakeup_efd_arm(wakeup);
-            if (status != UCS_OK) {
+            if (UCS_ERR_BUSY == status) {
+                result_status = status;
```

Comment:
@yosefe Yes, you right, but I didn't return UCS_OK if some transport return UCS_ERR_BUSY. I just walk over all transports, call arm() for each of them and return UCS_ERR_BUSY if **anyone** return this status.